plugins/node/opentelemetry-instrumentation-mysql/src/instrumentation.ts

```diff
@@ -343,4 +366,43 @@ export class MySQLInstrumentation extends InstrumentationBase<
       };
     };
   }
+  private _setPoolcallbacks(
+    pool: mysqlTypes.Pool | mysqlTypes.PoolCluster,
+    thisPlugin: MySQLInstrumentation
+  ) {
+    pool.on('connection', connection => {
+      thisPlugin._connectionsUsage.add(1, {
+        'db.client.connection.usage.state': 'idle',
```

Comment:
The spec defines the attribute name to be `state`, without the prefix `db.client.connection.usage.`. Am I interpreting the spec correctly?

Reply:
see the spec [here](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/database-metrics.md#connection-pools), it says: _"All `db.client.connections.usage` measurements MUST include the following attribute"_. I understood it as if the field `state` should be a sub-field and include this prefix but maybe I didn't understand correctly... who can determine this issue?

Reply:
Kind note: please avoid marking threads as resolved before we settled the final resolution. Thank you!

I don't find any words that these attribute names are abbreviated. Other attribute definition shows that if they need to be scoped, they are cleared spelled out: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/rpc-metrics.md#attributes.

@lmolkova As the author of the spec change, would you mind sharing your insight on this? Should the attribute name be scoped with `db.client.connection.usage`?

Reply:
It seems you're right. (if they want to scope it - they say it explicitly). I'll change it.

Reply:
> @lmolkova As the author of the spec change, would you mind sharing your insight on this? Should the attribute name be scoped with db.client.connection.usage

@legendecas  you probably wanted to summon @mateuszrzeszutek (who wrote DB metrics spec). If I read it correctly. `state` seems to be an attribute name and it doesn't have a namespace.